drivers/adp536x/adp536x.c

```diff
@@ -112,6 +113,10 @@
 #define ADP536X_BAT_OC_CHG_DGT_OC_CHG_MSK		GENMASK(4, 3)
 #define ADP536X_BAT_OC_CHG_DGT_OC_CHG(x)		(((x) & 0x03) << 3)
 
+/* Buck configure register. */
+#define ADP536X_BUCK_CFG_DISCHG_BUCK_MSK		BIT(1)
+#define ADP536X_BUCK_CFG_DISCHG_BUCK(x)			(((x) & 0x01) << 1)
```

Comment:
nit: fix indentation here

Reply:
Hm, it looks fine in my editor with tab size of 8. Could it be a Github thing? Also here it looks okay: https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/714/files/ee4cb25a4a7e7d6abfc91e03d3af281a94929794#diff-74af8e9210e9c301c5bb09318c480909

Reply:
Yes. Apparently Github still incorrectly handles tabs in the split diff. When you [view](https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/ee4cb25a4a7e7d6abfc91e03d3af281a94929794/drivers/adp536x/adp536x.c#L118) the whole file or switch to the unified diff, everything looks fine (these two views must have been corrected quite recently, earlier I got used to see Github incorrectly displaying indentation made with tabs in specific cases, probably because one character, like `+` or `-`, added before the actual content of a given line).

Reply:
Yeap, that is strange. Github shows this un-aligned, because line 118 has 3 tabs, compared to line 117 which has 2. But my editor shows, 2 tabs on each line.